tests/testsuite.src/run_misc.at

```diff
@@ -10921,7 +10919,8 @@ END OF DUMP - prog
 
 # AT_DATA workaround via sed:
 AT_CHECK([$SED -e 's/_$//' reference_fdls_tmpl > reference], [0], [], [])
-AT_CHECK([$SED -e 's/compiled ... .. .... ..:..:../compiled MMM DD YYYY HH:MM:SS/g' \
+AT_CHECK([$SED -e 's/compiled ... .. .... ..:..:../compiled MMM DD YYYY HH:MM:SS/g;
+                   s/Started by [[^[:cntrl:]]]*prog_fdls.exe/Started by .\/prog_fdls/g' \
```

Comment:
`[[^[:cntrl:]]]` is likely not portable, but at least here and below we can use `.*`, too.